trie/trie_test.go

```diff
@@ -204,7 +204,7 @@ func buildSmallTrie() *Trie {
 	for _, test := range tests {
 		err := trie.Put(test.key, test.value)
 		if err != nil {
-			return nil
+			t.Fatal(err)
```

Comment:
Note: this is a good place for a `panic` if you don't wanna pass around `t`

Reply:
hmm I will consider, I like passing around `t` though

Reply:
you => :smiling_imp: